### ISSUE TITLE
CA-Certificate validation error when cofiguring email via SMTPs

### ISSUE THREAD

**Issue by sudoleg** [fjogeleit/policy-reporter#404]
I installed PolicyReporter via a Helm Chart and am currently trying to send policy reports via email using smtps (encryption starttls). The logs of the "summary-report" Pod show this error at the end:
```
{"error": "Mail Error on STARTTLS: tls: failed to verify certificate: x509: certificate signed by unknown authority"}
``` 
The error message suggests that I have to install the CA certificates of my company. I tried to mount the certificate (.pem-file)  under `/etc/ssl/certs` and `/usr/local/share/ca-certificates` but I still get the error. Any Ideas what I doing wrong?

thanks in advance :)

**Comment by fjogeleit** [comment]
Hey, the email client configuration needs to be extended to be able to handle custom certificates. I'll take a look on it.

**Comment by fjogeleit** [comment]
with the latest release you can configure your cert as extraVolume and configure the path in the SMTP settings as `certificate`.

**Comment by sudoleg** [comment]
Hey @fjogeleit , thank you for the veryquick response and fix! Unfortenately, the indentation seems to be wrong. I've fixed this in PR #407
